tortoise/models.py

```diff
@@ -192,6 +192,30 @@ def __new__(mcs, name: str, bases, attrs: dict, *args, **kwargs):
         meta_class = attrs.get("Meta", type("Meta", (), {}))
         pk_attr = "id"
 
+        # Searching for Field attributes in the class hierarchie
+        def __search_for_field_attributes(base, attrs: dict):
+            """
+            Searching for class attributes of type fields.Field
+            in the given class.
+
+            If an attribute of the class is an instance of fields.Field,
+            then it will be added to the attrs dict. But only, if the
+            key is not already in the dict. So derived classes have a higher
+            precedence. Multiple Inheritence is supported from left to right.
+
+            After checking the given class, the funkction will look into
```

Comment:
I think `funkction` is typo for `function`?

Reply:
No. intention. Just kidding. I will correct it of course. Anyway, this was not meant as the final version. Just a wip to get feedback on detail level and writing style. Is it understandable, etc. Thanks for finding the typo.